CHANGES.md

```diff
@@ -12,6 +12,8 @@
 - Duplicate and move files and projects [Taiga #267](https://tree.taiga.io/project/penpot/us/267)
 - Import SVG will create Penpot's shapes
 - Improve french translations [#731](https://github.com/penpot/penpot/pull/731)
+- Replace Slate-Editor with DraftJS [#752](https://github.com/penpot/penpot/pull/752)
```

Comment:
Woudn't it be better to link to the US? https://tree.taiga.io/project/penpot/us/1346

Reply:
I link it on the PR description